src/server/app.js

```diff
@@ -28,12 +28,15 @@ if (process.env.NODE_ENV === 'production-notugly') {
   )
 }
 
-app.use(express.static(path.resolve(__dirname, '../../', 'build')))
-app.get('/embed/:id', videoRoute)
-app.get('/play/:id', videoRoute)
+// app.use(express.static(path.resolve(__dirname, '../../', 'build')))
```

Comment:
Why is this removed? I think this would break a lot

Reply:
i've removed because serves also index.html, do you know how to fix that?

Reply:
Without this I would not expect any of our asset loading to work (i.e. images will not load).

I'm sure there is an `express` api for excluding certain files. However, why do we need to exclude `/index.html`?

Reply:
because it wouldn't get in the get request. I'll check with the handlebar features.